js/main.js

```diff
@@ -17,8 +17,8 @@ function updateImage(event) {
 function submit(event) {
   event.preventDefault();
   var object = { title: title.value, url: photoURL.value, notes: notes.value };
-  object.EntryId = data.EntryId;
-  data.EntryId++;
+  object.entryId = data.entryId;
```

Comment:
Update the property name you are using from the data model after you fix it from my other note

`object.entryId = data.nextEntryId`